xarray/core/formatting.py

```diff
@@ -1051,7 +1051,10 @@ def diff_datatree_repr(a: DataTree, b: DataTree, compat):
 def _single_node_repr(node: DataTree) -> str:
     """Information about this node, not including its relationships to other nodes."""
     if node.has_data or node.has_attrs:
-        ds_info = "\n" + repr(node._to_dataset_view(rebuild_dims=False))
+        # TODO: change this to inherited=False, in order to clarify what is
+        # inherited?
```

Comment:
This is basically the same TODO that we mentioned in https://github.com/pydata/xarray/issues/8572#issuecomment-2218020742.

Reply:
Raised https://github.com/pydata/xarray/issues/9464 to track this.

Reply:
I made https://github.com/pydata/xarray/issues/9463 for this

Reply:
Actuall #9463 is better!